edge/pkg/metamanager/process.go

```diff
@@ -293,13 +293,13 @@ func (m *metaManager) processRemoteQuery(message model.Message) {
 			string(metaManagerConfig.Config.ContextSendModule),
 			message,
 			time.Duration(metaManagerConfig.Config.RemoteQueryTimeout)*time.Second)
-		klog.Infof("########## process get: req[%+v], resp[%+v], err[%+v]", message, resp, err)
 		if err != nil {
-			klog.Errorf("remote query failed: %v", err)
+			klog.Errorf("remote query failed, req[%s], err: %v", msgDebugInfo(&message), err)
 			feedbackError(err, "Error to query meta in DB", message)
 			return
 		}
 
+		klog.Infof("process remote query: req[%s], resp[%s]", msgDebugInfo(&message), msgDebugInfo(&resp))
```

Comment:
Seems that this log is better for debuging?
```suggestion
		klog.V(4).Infof("process remote query: req[%s], resp[%s]", msgDebugInfo(&message), msgDebugInfo(&resp))
```

Reply:
> Seems that this log is better for debuging?

Done.